MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -553,6 +585,20 @@ private async Task<string> ExtractImageInternal(string inputPath, string contain
 
             var mapArg = imageStreamIndex.HasValue ? (" -map 0:v:" + imageStreamIndex.Value.ToString(CultureInfo.InvariantCulture)) : string.Empty;
 
+            var enableHdrExtraction = allowTonemap && videoStream != null && !string.IsNullOrEmpty(videoStream.VideoRange) && string.Equals(videoStream.VideoRange, "HDR", StringComparison.OrdinalIgnoreCase);
```

Comment:
If i'm reading this correctly, this can be changed to

```suggestion
            var enableHdrExtraction = allowTonemap && string.Equals(videoStream?.VideoRange, "HDR", StringComparison.OrdinalIgnoreCase);
```
String.Equals permits null as a paramter, so will return false if videorange is null